test/Microsoft.AspNetCore.SignalR.Tests/HubEndpointTests.cs

```diff
@@ -36,7 +37,7 @@ public class HubEndpointTests
 
             using (var client = new TestClient())
             {
-                var endPointTask = endPoint.OnConnectedAsync(client.Connection);
+                Task endPointTask = await client.ConnectAsync(endPoint);
```

Comment:
Does var not work here?

Reply:
`dynamic` is just the worst.

Reply:
Yea, we can fix that by making HubEndPoint\<T\> derive from  EndPoint